pegasus/sites.v3/code.org/views/about_donors_image.haml

```diff
@@ -11,18 +11,24 @@
     text-align: center;
   }
 
+// Size large is a 2 column grid with larger height for larger logos
+// Size medium is a 3 column grid with medium height for logos
+// Size small is a 3 column grid for small logos or name only
+- columns = size == "large" ? 2 : 3
+- height = size == "large" ? 150 : size == "medium" ? 200 : 100
+- fontsize = size == "small" ? "18" : "14"
+
 %table
-  - supporters.each_slice(columns).with_index do |group, groupindex|
+  - supporters.each_slice(columns) do |group|
```

Comment:
In that case it would just be a series of `float: left` divs one after another, and they'd wrap.  Only occurred to me so we didn't have to worry about managing this splicing in the HAML side, but could be tricky to get the borders looking right, so not a big deal.